code/modules/assembly/logic/math.dm

```diff
@@ -171,100 +173,153 @@ var/global/list/math_circuit_operations_list = list("ADD", "SUBTRACT", "MULTIPLY
 /obj/item/device/assembly/math/get_value(value)
 	if(!values.len) return 0
 
-	if(value != "Result")
+	if(value != "Result" && value != "Variables")
```

Comment:
These comparisons should be using string defines because consistency is required.